css/main.css

```diff
@@ -1,3 +1,320 @@
 /* Add your CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+@import url('https://fonts.googleapis.com/css2?family=Frank+Ruhl+Libre:wght@300&family=Nanum+Gothic&family=Sarabun:wght@100&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Frank+Ruhl+Libre:wght@300&family=Nanum+Gothic&family=Sarabun:wght@100&display=swap');
+
+ * {
+    padding: 0px;
+    margin: 0px;
+ }
+body {
+    background:linear-gradient(
+      to top,
+      #E6DACD 0%,
+      #E6DACD 60%,
+      #fff 60%,
+      #fff 100%  
+    );
+    width: 390px;
+    height: 844px;
+  
+ align-content: center;
+ text-align: center;
+
+  }
+  /* header text */
+  .scouer{
+    margin-left: 15px;
+    /* padding: 2px; */
+    background-color: #0150FD;
+    width: 16px; 
+     height: 16px;
+     margin-top: 10px;
+  }
+  .h1_1{
+    font-family: 'Poppins', sans-serif;
+  }
+  .text_nav1{
+    margin-left: 5px;
+    display: flex;
+    font-family: 'Frank Ruhl Libre', serif;
+    
+    
+  }
+  .text_nav2{
+    margin-left: 16px;
+    font-family: 'Sarabun', sans-serif;
+  }
+  #toggle {
+    display: none;
+  }
+
+ 
+  /**
+    Hamburger
+  **/
+  .hamburger {
+    position: absolute;
+    top: 3em;
+    right: 5%;
+    margin-left: -2em;
+    margin-top: -45px;
+    width: 2em;
+    height: 45px;
+    z-index: 5;
+  }
+  
+  .hamburger div {
+    position: relative;
+    width: 3em;
+    height: 7px;
```

Comment:
Again also for the height don't use px is not the best practice use  a bit units like vh or % please!

Reply:
I think sometimes PX can be ok for height of smaller elements like `icons` and the like, which generally have a pretty consistent height across screen sizes, but agreed with Altom, for larger layout elements, `vh` is preferred, I generally avoid `%` for height as somewhere higher in the tree an element will require a defined height in `vh` or `px`